examples/layouting.jl

```diff
@@ -486,4 +486,95 @@ end
 
     end
 
+    @cell "markers, an arraow, and ellipses in CairoMakie" [layout] begin
+
+        using DataFrames, Distributions, Colors
+        using CairoMakie, MakieLayout
```

Comment:
Is CairoMakie a necessity here?  Ideally, this should be run in GLMakie also.

Reply:
no no, just figured that since this whole thing came to be from CairoMakie it would be cool to do that. I realize that it's better to decouple the need for a specific backend. Cool.